### PR TITLE
Script to generate Kafka keystores for SSL

### PR DESCRIPTION
**Pull request by nasark**
- script to generate kafka truststore and keystore files using provided CA (such as `root.crt` created in cert_generator.rb) and keystore password determined by user

Needed for:
- https://github.com/ManageIQ/manageiq-documentation/pull/1728

@miq-bot assign @bdunne 
@miq-bot add_reviewer @Fryguy 
@miq-bot add_label enhancement

### CODE REVIEW

File: tools/keystore_generator.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  exit 1
+fi
+
+if [ -z "$KEYSTORE_PASS" ]; then
+  echo "Please provide a keystore password in KEYSTORE_PASS"
+  exit 1
+fi
+
+# Generate truststore containing CA
+keytool -keystore ./kafka.truststore.jks \
```
Comment (Fryguy) [inline tools/keystore_generator.sh:20]: Where does `keytool` come from?

File: tools/keystore_generator.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+if [ ! -e "$CA_CERT_PATH" ] || [ ! -e "$CA_KEY_PATH" ]; then
+  echo "CA does not exist, please provide the corrrect paths in CA_CERT_PATH and CA_KEY_PATH"
+  exit 1
+fi
+
+if [ -z "$KEYSTORE_PASS" ]; then
+  echo "Please provide a keystore password in KEYSTORE_PASS"
+  exit 1
+fi
```
Comment (Fryguy) [inline tools/keystore_generator.sh:17]: For security reasons we may want to read the password from STDIN.

File: tools/keystore_generator.sh (hunk 1 above)
Comment (nasark) [reply tools/keystore_generator.sh:20]: `keytool` comes with any Java distribution. I guess this means that the script assumes the user has Java on their system. Would this be an issue?

**Comment by nasark** [comment]
> Is it possible to go the other direction leave generation of the certificates in the current script and have an entrypoint script on the kafka container that creates a keystore and adds those certs to it?

https://github.com/ManageIQ/container-kafka/pull/2

**Comment by kbrock** [comment]
Would have preferred this being in appliance_console{_cli} but if that does not create certificates, then that request is out of scope.

Did we want the bot to close this?
Seems it is still outstanding

**Comment by nasark** [comment]
This can be closed as we are no longer moving forward with Bitnami and this will not be needed for Strimzi

**Pull request closed without merging**